{"name": "test_job_cities[vetro]", "status": "failed", "statusDetails": {"message": "AssertionError: Test found mismatching job cities for the following jobs: ['Reprezentant Medical Veterinar', 'Reprezentant Medical Veterinar']", "trace": "self = <TestJobsValidator.test_api.TestScrapers object at 0x7fa525030730>\nsetup = (['Key Account Manager, divizia Ferme', 'Reprezentant Medical Veterinar', 'Reprezentant Medical Veterinar', 'Reprezent...Olt', 'Valcea'], ['Cluj', 'Bistrita-Nasaud'], ['Mures', 'Alba'], ['Botosani', 'Iasi'], ['Bucuresti', 'Cluj', 'Timis']])\n\n    @pytest.mark.smoke\n    def test_job_cities(self, setup):\n        \"\"\"\n        Test job titles contain special characters\n        \"\"\"\n        peviitor_jobs_data = setup\n        job_titles = peviitor_jobs_data[0]\n        job_cities = peviitor_jobs_data[1]\n>       TestScrapers.setup_tests.jobdetails.check_job_cities(job_cities, job_titles)\n\ntests/TestJobsValidator/test_api.py:80: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/TestJobsValidator/api_utils/job_details_utils.py:124: in check_job_cities\n    self.cityutils.check_job_cities(job_cities, job_titles)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.TestJobsValidator.api_utils.job_cities_utils.CitiesTestUtils object at 0x7fa5256224a0>\nactual_cities = [['Brasov', 'Sibiu', 'Targu-Mures', 'Bistrita', 'Cluj-Napoca'], ['Craiova', 'Slatina', 'Ramnicu Valcea'], ['Cluj-Napoc...istrita', 'Maramures'], ['Mures', 'Alba', 'Harghita'], ['Botosani', 'Iasi'], ['Bucuresti', 'Cluj-Napoca', 'Timisoara']]\njob_titles = ['Key Account Manager, divizia Ferme', 'Reprezentant Medical Veterinar', 'Reprezentant Medical Veterinar', 'Reprezentant Medical Veterinar', 'Reprezentant Medical Veterinar', 'Reprezentant tehnic È™i promovare']\n\n    def check_job_cities(self, actual_cities, job_titles):\n        # Itterate over actual cities and get_proper_city > city list or none if one was not found\n    \n        msg = \"Unknown error occured\"\n    \n        missing_cities = []\n        missing_cities_job_titles = []\n    \n        for city_list, job_title in zip(actual_cities, job_titles):\n            if not get_proper_city(city_list):\n                missing_cities.append(city_list)\n                missing_cities_job_titles.append(job_title)\n    \n        if missing_cities:\n            msg = f\"Test found mismatching job cities for the following jobs: {missing_cities_job_titles}\"\n    \n        if not actual_cities and not job_titles:\n            msg = f\"Cannot grab any job cities or job titles from the validator\"\n            allure.step(msg)\n            raise AssertionError(msg)\n    \n        allure.step(msg)\n>       assert missing_cities == [], msg\nE       AssertionError: Test found mismatching job cities for the following jobs: ['Reprezentant Medical Veterinar', 'Reprezentant Medical Veterinar']\n\ntests/TestJobsValidator/api_utils/job_cities_utils.py:30: AssertionError"}, "description": "\n        Test job titles contain special characters\n        ", "parameters": [{"name": "scraper_class", "value": "'vetro'"}], "start": 1732513279869, "stop": 1732513279879, "uuid": "c3d4d106-1814-4051-ae13-ef0bb332b66d", "historyId": "af8666a95b0a912b068d1c9887804bdb", "testCaseId": "1204388ca11ed5cbe72e4cae6999b749", "fullName": "tests.TestJobsValidator.test_api.TestScrapers#test_job_cities", "labels": [{"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "tests.TestJobsValidator"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestScrapers"}, {"name": "host", "value": "fv-az1425-114"}, {"name": "thread", "value": "1874-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.TestJobsValidator.test_api"}]}